coordinator/migrations/2023-09-04-090152_add_order_state_and_matches/up.sql

```diff
@@ -6,7 +6,10 @@ CREATE TYPE "OrderState_Type" AS ENUM (
     'Failed'
 );
 
-ALTER TABLE "orders" ADD COLUMN "order_state" "OrderState_Type" NOT NULL DEFAULT 'Open';
+ALTER TABLE "orders"
```

Comment:
Same as in the other commit, please do not update old migration files. This can lead to annoying errors.

Reply:
I've split up the migration scripts into dedicated folders per commit.